createManagedMasterK8s.groovy

```diff
@@ -93,7 +93,9 @@ newInstance.setDisplayName(masterDisplayName)
  * CONFIGURATION BUNDLE *
  ************************/
 if(cascBundle?.trim()) {
+    // Note: ConnectedMasterCascProperty supported since 2.277.4.x. For earlier version, comment out the following.
```

Comment:
Probably also worth a comment that the following two lines are required to be in the strict order they are currently.

Reply:
I didn't dig into why, but It broke for me when using them in the opposite order. /shrug